config/jobs/kubernetes/sig-arch/kubernetes-depstat.yaml

```diff
@@ -0,0 +1,24 @@
+presubmits:
+  kubernetes/kubernetes:
+  - name: kubernetes-depstat
+    decorate: true
+    always_run: true
+    skip_report: true
+    spec:
+      containers:
+      - image: golang
+        command: ["/bin/bash"]
+        args: ["-c", "apk update; apk add git; 
+          set -euo; 
+          BASE_SHA="${PULL_BASE_SHA:-""}"; ARTIFACTS="${ARTIFACTS:-tmp/artifacts}";
+          mkdir -p "${ARTIFACTS}";
+          export GO111MODULE=on;
+          go get github.com/kubernetes-sigs/depstat;
```

Comment:
`go get` or `go install`?

Reply:
were you able to test this locally? you can use `docker run` with the `golang` image drop into shell and simulate the script. you can volume mount kubernetes/kubernetes as well.